handbook/company/pricing-features-table.yml

```diff
@@ -660,28 +664,8 @@
   waysToUse:
     - description: Support ACME as a protocol for MDM certificate generation. Coming soon (2024-12-31) #customer-rosner
       moreInfoUrl: https://github.com/fleetdm/fleet/issues/15611 
-- industryName: Low-level MDM commands for macOS, iOS/iPadOS*, and Windows (e.g. remote restart)
```

Comment:
Consolidated with "Enforce OS settings." Just realized that I also need to consolidate the ways to use and descriptions too. Will do that in another commit.